api/valor_api/main.py

```diff
@@ -233,13 +240,17 @@ def get_prediction(
     status_code=200,
     dependencies=[Depends(token_auth_scheme)],
     tags=["Labels"],
-    description="Fetch labels using optional JSON strings as query parameters.",
+    description="Fetch all labels.",
 )
 def get_labels(
     response: Response,
-    filters: schemas.FilterQueryParams = Depends(),
-    offset: int = 0,
-    limit: int = -1,
+    offset: int = Query(
+        0, description="The start index of the items to return."
+    ),
+    limit: int = Query(
```

Comment:
is there a precedent for using this design pattern? saying limit is an int, but then defaulting it to a `Query` object seems very unnatural to me. I'm hoping this is a common pattern you can point me to in other repos

Reply:
This is a fastapi defintion to specify that an arg is a query parameters. [Link](https://fastapi.tiangolo.com/tutorial/query-params-str-validations/#add-query-to-annotated-in-the-q-parameter)

Reply:
gotcha, thank you! I was confused because we used to have an object named `Query` too, but I think you got rid of it recently